internal/version/version.go

```diff
@@ -8,4 +8,4 @@ package version
 // Tag specifies the current release tag. It needs to be manually
 // updated. A test checks that the value of Tag never points to a
 // git tag that is older than HEAD.
-const Tag = "v1.37.0"
+const Tag = "v1.38.0"
```

Comment:
This is good, but someone else already merged this change in the mean time. If you rebase, you should be good to go without this.